agent/lib_aws_sdk_php.h

```diff
@@ -111,6 +136,40 @@ void nr_aws_sdk_mshutdown(void);
 extern char* nr_lib_aws_sdk_php_get_command_arg_value(char* command_arg_name,
                                                       NR_EXECUTE_PROTO);
 
+/*
+ * Purpose : Handle when a DynamoDbClient initiates a command
+ *
+ * Params  : 1. segment : end the segment as a datastore segment
+ *           2. command_name_string : the string of the command being called
+ *           3. command_name_len : the length of the command being called
+ *           4. NR_EXECUTE_ORIG_ARGS (execute_data, func_return_value)
+ *
+ * Returns :
+ *
+ */
+extern void nr_lib_aws_sdk_php_dynamodb_handle(nr_segment_t* segment,
+                                               char* command_name_string,
+                                               size_t command_name_len,
+                                               NR_EXECUTE_PROTO);
+
+/*
+ * Purpose : Populate DynamoDbClient datastore_params and cloud_attrs.  This
+ will extract region and host/port and set the value in the appropriate struct.
+             Note: A host/port other than the AWS DynamoDb default is very
+ uncommon and used only for development environments
+ *
+ * Params  : 1. datastore_params : datastore params to modify
+ *           2. cloud_attrs : cloud attributes to modify
+ *           3. NR_EXECUTE_ORIG_ARGS (execute_data, func_return_value)
+ * Returns :
+ * Note: caller is responsible for freeing the cloud_attrs->cloud_resource_id
+ and datastore_params->collection and datastore_params.instance.port_path_or_id
+ */
```

Comment:
It looks like the formatter did a number on this section. I tried to fix it on my machine but you may want to double check it yourself:
```suggestion
/*
 * Purpose : Populate DynamoDbClient datastore_params and cloud_attrs.  This
 *           will extract region and host/port and set the value in the
 *           appropriate struct.
 *
 *           Note: A host/port other than the AWS DynamoDb default is very
 *           uncommon and used only for development environments
 *
 * Params  : 1. datastore_params : datastore params to modify
 *           2. cloud_attrs : cloud attributes to modify
 *           3. NR_EXECUTE_ORIG_ARGS (execute_data, func_return_value)
 * Returns :
 * Note: caller is responsible for freeing the cloud_attrs->cloud_resource_id
 *       and datastore_params->collection and
 *       datastore_params.instance.port_path_or_id
 */

```